README.md

```diff
@@ -72,54 +74,56 @@ cp .env.example .env
 npm run dev
 ```
 
-### Build and Run
+#### Build and Run
 
 - To build the project for production (to generate static files for hosting)
 
 ```bash
 npm run build
 ```
-## Docker: Local Development
+### Docker: Local Development
 
 **Prerequisites:**
 
 - [Docker](https://docs.docker.com/engine/install/)
 
-### Clone the repository
+#### Clone the Repository
 
 ```
 git clone git@github.com:arcana-network/demo-app.git
 ```
 
-### Setup Container Environment
+#### Setup Container Environment
 
 ```
 cp .env.example .env
 ```
 
-### Build and Run
+#### Build and Run
 
 Run local environment with demo-app
 
 ```
 make run-local
 ```
-# 📚 Documentation
+## 📚 Usage
+
+Check out the [Demo App Usage Guide](https://docs.beta.arcana.network/docs/demo-app) for usage details.
```

Comment:
Do you recommend linking to the production docs instead of the beta docs here?

Reply:
I guess so - will update

Reply:
```suggestion
Check out the [Demo App Usage Guide](https://docs.arcana.network/docs/demo-app) for usage details.